src/main/java/io/mosip/digitalcard/service/DigitalCardService.java

```diff
@@ -13,5 +13,5 @@ public interface DigitalCardService {
 
     DigitalCardStatusResponseDto getDigitalCard(String rid);
 
-    boolean createDigitalCard(EventModel eventModel);
+    boolean initiateCredentialRequest(EventModel eventModel);
```

Comment:
we are printing in the log.

Reply:
The log says success and we print the boolean without any reason for error, it is not making sense. In what case this boolean is going to return as false, since my understanding is if there is a situation of failure, we throw back a exception and exception handler is going to return the proper error back in response.